flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/TemporalTypesTest.scala

```diff
@@ -945,6 +1030,14 @@ class TemporalTypesTest extends ExpressionTestBase {
       "1437699600")
   }
 
+  @Test
+  def test(): Unit = {
```

Comment:
change name

Reply:
forgot to remove useless test case. 
already covered in testHighPrecisionTimestamp